packages/orbit-components/src/primitives/MobileDialogPrimitive/__tests__/index.test.jsx

```diff
@@ -18,15 +18,17 @@ afterAll(() => {
 });
 describe("MobileDialogPrimitive", () => {
   it("should have expected DOM output", () => {
+    const onShown = jest.fn();
     render(
       // eslint-disable-next-line jsx-a11y/tabindex-no-positive
-      <MobileDialog tabIndex="1" dataTest="test" content="My text link">
+      <MobileDialog tabIndex="1" dataTest="test" onShow={onShown} content="My text link">
         children
       </MobileDialog>,
     );
     const children = screen.getByText("children");
     expect(children).toHaveAttribute("tabindex", "1");
     userEvent.click(children);
+    expect(onShown).toHaveBeenCalled();
```

Comment:
I think we can be more precise here and assert that it was called only once. By doing this, we are asserting that the function is called, but it can be called for whatever reason, or even multiple times. If we assure it is only called once, at least we can know it is not called on every state update. WDYT?

Reply:
makes sense, fixed